src/api.js

```diff
@@ -4,4 +4,10 @@ export default class API {
   static getMovies() {
     return Axios.get('http://localhost:3000/api/v1/movies');
   }
+
+  static getData(searchFor, searchBy) {
+    return Axios
+      .get('http://localhost:3000/api/v1/movies')
+      .then(movies => movies.data.filter(movie => movie[searchBy] === searchFor));
```

Comment:
api should only make requests and return responses. Nothing more. All logis should be handled by component